web/assets/css/app.css

```diff
@@ -341,3 +341,8 @@ table.table-samples td {
     background: lightgrey;
     cursor: pointer;
 }
+
+.disableClick {
+    pointer-events: none;
+    cursor: default;
+}
```

Comment:
I didn't know this was a reliable way of removing click events from an element. Guessing there's nothing in Bootstrap itself that does this?